includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Requires_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Requires_Javascript_Trait {
+
+	/**
+	 * Display fallback message when Javascript is disabled
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string noscript HTML tag,
+	 */
+	protected function get_noscript_html() {
+
+		ob_start();
```

Comment:
I think the usage of an output buffer here is unnecessary. We could just call this method `render_noscript_html()` instead. Everywhere this method is used we immediately `echo` it, so no need for this.